### PR TITLE
Added page with many grunt options

### PR DESCRIPTION
**Pull request by tedivm**

### CODE REVIEW

File: source/contributed/advanced_grunt.md [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+
+A common frustration amongst new players is that folders are not available by default. This can be changed using Grunt.
+
+To get started install the "grunt-contrib-copy" and "grunt-contrib-clean" plugins.
+
+    npm install grunt-contrib-clean --save-dev
+    npm install grunt-contrib-copy --save-dev
+
+
+In this case the "copy" plugin is going to be used to move code from the `src` directory to `dist`. The plugin as an option to rename files, so a function to convert directory delimiters (slashes) to underscores is used to flatten the file structure.
```
Comment (artch) [inline source/contributed/advanced_grunt.md]: Would be nice to see here an example of modules and directory structure "before" and "after".

**Comment by tedivm** [comment]
All feedback taken into account with the latest update.

**Comment by tedivm** [comment]
I'll take the subscription tokens to my account, and a link to my github account would be great. Thanks!

**Comment by artch** [comment]
Article [published](http://docs.screeps.com/contributed/advanced_grunt.html). Thanks for contribution!

**Pull request merged**